packages/plugins/ui-theme/src/components/Icons/DevsIcons/ES6Module.tsx

```diff
@@ -0,0 +1,13 @@
+import styled from '@emotion/styled';
+import { Theme } from '@mui/material';
+import React from 'react';
+
+import icon from './es6modules.svg';
+
+const ImgIcon = styled.img<{ theme?: Theme }>(({ theme }) => ({
+  marginLeft: theme?.spacing(1),
+}));
+
+export function ES6Modules() {
```

Comment:
In general, I would vote for a name change to `ESModules` and `ESM` in the SVG.

Reply:
I'll rename it thanks !